Gems/Camera/Code/Source/ViewportCameraSelectorWindow.cpp

```diff
@@ -98,22 +97,32 @@ namespace Camera
 
         void CameraListModel::OnCameraAdded(const AZ::EntityId& cameraId)
         {
-            // If the camera entity is not an editor camera entity, don't add it to the list.
-            // This occurs when we're in simulation mode.
-
-            //We reset the m_firstEntry value so we can update m_lastActiveCamera when we remove from the cameras list
-            m_firstEntry = true;
+            // if the camera entity is not an editor camera entity, don't add it to the list
+            // this occurs when we're in simulation mode.
```

Comment:
Nit: The capitalization / punctuation was actually correct before the change. Not sure why the "if" and "this" became decapitalized and the period after "list" was removed.

Reply:
I routinely do not bother with capitalization and periods in .cpp files and reserve that style for API comments - this style is prevalent in quite a few files (to indicate guide comments) and as I'm now the de-facto maintainer of this file I updated the style while making the other edits/changes - I'd prefer to leave as-is if you don't mind